src/blob-store/live-download.js

```diff
@@ -241,7 +242,7 @@ export class DriveLiveDownload extends TypedEmitter {
       const [blobs] = await once(this.#drive, 'blobs', { signal: this.#signal })
       return blobs.core
     } catch (e) {
-      if (e.name === 'AbortError') return
+      if (isRecord(e) && e.name === 'AbortError') return
```

Comment:
instead of relying on the helper, maybe something like this would suffice?

```suggestion
      if (e instanceof Error && e.name === 'AbortError') return
```